st2tests/tests/actions/render.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
The tests pack I'm asking you to modify is located @ https://github.com/StackStorm/st2tests/tree/master/packs/tests.  It is a separate repo.  You're adding files in the wrong place.

Reply:
Sorry about this, did not realize there is a different repo for `st2tests`.  On cicd for integration test, does `tests` pack installed on the same directory as https://github.com/StackStorm/st2/tree/master/st2tests/tests? Because it needs to be copied to `/opt/stackstrom/packs`, need to know the source location.

Since i don't understand how https://github.com/StackStorm/st2tests/tree/master/packs/tests is used, for earlier unit test submission, `dummy_pack_7` is used for testing, should we change it to the same `tests` pack?